app/src/main/java/com/rootstrap/android/ui/activity/main/targetpoint/CreateTargetView.kt

```diff
@@ -66,15 +134,41 @@ class CreateTargetView(
         return isTopicValid
     }
 
-    fun expandCollapseSheet() {
+    fun expandCollapseCreateTargetSheet() {
         if (bottomSheetBehavior.state != BottomSheetBehavior.STATE_EXPANDED) {
             bottomSheetBehavior.state = BottomSheetBehavior.STATE_EXPANDED
         } else {
             bottomSheetBehavior.state = BottomSheetBehavior.STATE_COLLAPSED
         }
```

Comment:
```suggestion
        bottomSheetBehavior.state = if (bottomSheetBehavior.state != BottomSheetBehavior.STATE_EXPANDED) {
            BottomSheetBehavior.STATE_EXPANDED
        } else {
            BottomSheetBehavior.STATE_COLLAPSED
        }
```

Reply:
From the Kotlin Coding Conventions:
prefer using **`if`** for binary conditions instead of when. 
Prefer using **`when`** if there are three or more options.